src/components/Item/Item.scss

```diff
@@ -1,28 +1,116 @@
-.cardContainer {
-    width: 200px;
-    height: 270px; 
-    margin: 10px; 
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700');
```

Comment:
Font should be the same on entire website, so it should be set in a "root" scss file, i.e. Application.scss